### PR TITLE
Prototype cleanup

### PR DESCRIPTION
**Pull request by stvnrlly**
Tasks:

- [x] Add tests
- [x] Add badges for tests, coverage, and code quality
- [x] Make sure that documentation is comprehensive
- [ ] Consider moving main site code out of `specialuseform`
- [x] Add more USWDS stuff

### CODE REVIEW

File: forestserviceprototype/specialuseform/views.py [hunk 1]
```diff
@@ -86,31 +99,38 @@ def change_application_status(request, permit_id, status):
         prototype.fr.cloud.gov/submitted/%s'.format(
```
Comment (lauraGgit) [inline forestserviceprototype/specialuseform/views.py]: Why do you have cloud.gov hard coded here?

File: forestserviceprototype/specialuseform/models.py [hunk 2]
```diff
@@ -3,49 +3,164 @@
 from django.contrib.auth.models import User
 from localflavor.us import models as localflavor
 
+
 # Create your models here.
 class Permit(models.Model):
```
Comment (lauraGgit) [inline forestserviceprototype/specialuseform/models.py:8]: Just making sure that with these changes that the primary key is now `permit.permit_id`?

File: forestserviceprototype/specialuseform/management/commands/create_permits.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+#! /usr/bin/env python3
+from django.core.management.base import BaseCommand, CommandError
+from specialuseform.factories import PermitFactory
+
+
+class Command(BaseCommand):
```
Comment (lauraGgit) [inline forestserviceprototype/specialuseform/management/commands/create_permits.py:6]: Can you add a bit in the readme about this.

File: forestserviceprototype/specialuseform/templates/specialuseform/includes/header.html [hunk 4]
```diff
@@ -0,0 +1,114 @@
+{% load static from staticfiles %}
+
+<header class="usa-header usa-header-extended" role="banner">
```
Comment (lauraGgit) [inline forestserviceprototype/specialuseform/templates/specialuseform/includes/header.html:3]: is there any way to load this in from the uswds package so we can more easily update? Probably not feasible, but just checking.

**Review by lauraGgit — COMMENTED** [review]
still need to run locally. to test.

File: forestserviceprototype/specialuseform/models.py (hunk 2 above)
Comment (stvnrlly) [reply forestserviceprototype/specialuseform/models.py:8]: It should be `permit.id`, the default. I didn't see a reason to replace it, but if there's a special format for the field I'm happy to add it back and document that.

File: forestserviceprototype/specialuseform/templates/specialuseform/includes/header.html (hunk 4 above)
Comment (stvnrlly) [reply forestserviceprototype/specialuseform/templates/specialuseform/includes/header.html:3]: Unfortunately, no, but there has been discussion in the organization of building a Django package to provide that, perhaps using https://github.com/department-of-veterans-affairs/django-designstandards. Some of the other projects have developed elements combining USWDS and Django features, so it'd be really cool to have.

File: forestserviceprototype/specialuseform/views.py (hunk 1 above)
Comment (stvnrlly) [reply forestserviceprototype/specialuseform/views.py]: Hmm, this is from a while back; let me see if I can update it real quick.

File: forestserviceprototype/specialuseform/management/commands/create_permits.py (hunk 3 above)
Comment (stvnrlly) [reply forestserviceprototype/specialuseform/management/commands/create_permits.py:6]: Done in https://github.com/18F/forest-service-prototype/pull/57/commits/2d451051b89d92962f4442f86b0498a134c5ec55.

**Comment by lauraGgit** [comment]
Locally found a few more things:
- [x] Until we have a need for a menu bar, I think we should maybe remove the dropdowns. It is a bit confusing
- [x] You still cannot submit without manually entering the second set of addresses for the `permit holder`
- [x] While it says its not required, you need to enter a spectator number or the permit breaks.
- [x] It might be nice to have a bit in the README on how to run the tests locally (pulled from the travis)
- [x] When run locally, the pytest has the following error `ValueError: The file 'css/uswds.min.css' could not be found with <whitenoise.storage.CompressedManifestStaticFilesStorage object at 0x10e182f28>.` Seems like `Whitenoise` might not be pulling in the third party files correctly when the tests are called.

**Pull request merged**